topics/admin/tutorials/ansible/tutorial.md

```diff
@@ -607,7 +607,7 @@ Now that you've built a small role, you can imagine that building real roles tha
 >    > > ```
 >    > > - downloading role 'git', owned by geerlingguy
 >    > > - downloading role from https://github.com/geerlingguy/ansible-role-git/archive/3.0.0.tar.gz
->    > > - extracting geerlingguy.git to /tmp/tmp.Jm681gks8d/roles/geerlingguy.git
+>    > > - extracting geerlingguy.git to /home/xhr/ansible/intro/roles/geerlingguy.git
```

Comment:
This should probably be ubuntu for the user, right?

Reply:
I wasn't sure we wanted to expose that username - I don't remember if we used it for attendees last time, or just trainers, so I kept `xhr`. So yes, let's use `ubuntu` then. I'll change it for all other occurrences too.

Reply:
done.